components/Response.js

```diff
@@ -47,51 +57,26 @@ class Response extends React.Component {
     return (
       <a href={linkTo} title={title}>
         <div>
-          <style jsx>
-            {`
-              .Response {
-                display: flex;
-                align-items: flex-start;
-                width: 100%;
-                margin: 10px;
-                max-width: 300px;
-                float: left;
-                position: relative;
-                height: 90px;
-                overflow: hidden;
-              }
-
-              .bubble {
-                padding: 0.25rem 1rem;
-              }
-
-              .name {
-                font-size: 1.5rem;
-              }
-
-              .description {
-                font-size: 1.2rem;
-              }
-
-              .star {
-                width: 14px;
-                height: 14px;
-                position: absolute;
-                top: 45px;
-                left: 0;
-              }
-            `}
-          </style>
-          <div className="Response">
-            {status === 'INTERESTED' && <object title={title} type="image/svg+xml" data={star} className="star" />}
+          <Container
+            display="flex"
+            alignItems="flex-start"
+            width="100%"
+            margin="10px"
+            maxWidth="300px"
+            float="left"
+            position="relative"
+            height="90px"
+            overflow="hidden"
+          >
+            {status === 'INTERESTED' && <StarObject title={title} type="image/svg+xml" data={star} className="star" />}
```

Comment:
From what I see in https://github.com/opencollective/opencollective-frontend/blob/a4bb9656cfbaccbb224922cdb6c5e91b39c52976/components/collective-page/sections/SponsorsAndParticipants.js#L61, it seems that this "INTERESTED" status is not used anymore. Let's take the opportunity to kill it :smiling_imp:

Reply:
Done 😄